django/contrib/postgres/lookups.py

```diff
@@ -60,3 +60,24 @@ def process_lhs(self, qn, connection):
             self.lhs = SearchVector(self.lhs)
         lhs, lhs_params = super(SearchLookup, self).process_lhs(qn, connection)
         return lhs, lhs_params
+
+
+class TrigramSimilar(PostgresSimpleLookup):
+    lookup_name = 'trigram_similar'
+    operator = '%%'
+
+
+class TrigramDistance(PostgresSimpleLookup):
```

Comment:
Unless I've misunderstood, a TrigramDistance lookup is the thing I want :) `a % b` is the same as `a <-> b < 0.7` but without any way to specify a different threshold. The example I gave in functions.txt is what I'd like – being able to filter objects by a provided input string, with a given threshold. Would that be possible with only the TrigramDistance expression?

Perhaps it could have a better name than `trigram_distance`; `(foo__trigram_similar="test")` is the same as `(foo__trigram_distance=("test", 0.7))`, something like `trigram_compare`? Dunno.